### DISCUSSION TITLE
about add channel

### DISCUSSION THREAD

**Discussion by mh292929** [iptv-org/community#264] [Q&A]
hi. i want contribute. i want add sum persian channel but i can not find logo. can i send channel name and channel url in comments and you add to playlist?

**Comment by keystroke3** [comment]
I think you can just add it here and the maintainers will do add it.

**Comment by mh292929** [comment]
must i send name and url in comments?

On Thu, Jul 22, 2021 at 2:06 PM Ted ***@***.***> wrote:

> I think you can just add it here and the maintainers will do add it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/iptv-org/iptv/discussions/3633#discussioncomment-1036823>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACCWHPI7OU6OX4B5BJJGE23TY7RCTANCNFSM5AZT5N5A>
> .
>


-- 
با سپاس, محمود هژبری

**Reply by BellezaEmporium** [reply]
Yes, send it here and we'll take care of it.

**Comment by mh292929** [comment]
hi. these are sum channels. please add and notify me, i'll share other channels.
line 1: it's channel name
line 2: it's category
line 3:  it is url
Bartar Tv
entertain
http://159.69.58.154/bartar/bartar.m3u8?token=HnKlfdRtgu76Lkg78Fds4vCFgt50LjgddWTCXgh6KJHG3VBNjh87AF
4U TV
entertain
http://116.202.255.113:1935/4utv/livehd/playlist.m3u8
Film 1
movies
http://159.69.58.154/film1/film1tv.m3u8?token=HnKlfdRtgu76Lkg78Fds4vCFgt50LjgddWTCXgh6KJHG3VBNjh87AF
Channel One
Political
https://2nbyjjx7y53k-hls-live.5centscdn.com/cls040317/0070c5b7ef083bdc8de09065c61a34d8.sdp/playlist.
bbc persian s2
news
http://159.69.58.154/bbc/master.m3u8
Iran International
news
https://dev-live.livetvstream.co.uk/LS-63503-4/index.m3u8

**Comment by mh292929** [comment]
sorry. last comment has sum problem.
please add these channels:
Bartar Tv
entertain
http://159.69.58.154/bartar/bartar.m3u8
4U TV
entertain
http://116.202.255.113:1935/4utv/livehd/playlist.m3u8
Film 1
movies
http://159.69.58.154/film1/film1tv.m3u8
Channel One
Political
https://2nbyjjx7y53k-hls-live.5centscdn.com/cls040317/0070c5b7ef083bdc8de09065c61a34d8.sdp/playlist.m3u8
bbc persian s2
news
http://159.69.58.154/bbc/master.m3u8
Iran International
news
https://dev-live.livetvstream.co.uk/LS-63503-4/index.m3u8

**Comment by mh292929** [comment]
Toonix
kids
http://51.210.227.134/hls/stream.m3u8
T2 Movies
movies
http://159.69.58.154/t2_movies/master.m3u8
PMC Royale
music
http://51.210.199.29/hls/stream.m3u8
Gem Series
movies
http://216.66.42.37:7777/GemSeries_HD.m3u8
NAVAHANG
music
http://51.210.227.130/hls/stream.m3u8

**Comment by mh292929** [comment]
FX
movies
http://51.210.199.52/hls/stream.m3u8
Ecran tv
entertain
http://51.210.199.40/hls/stream.m3u8
Irane Aryaee
Political
http://159.69.58.154/irane_arya/irane_arya.m3u8
avang tv
music
http://appavang.flashmediacast.com:1935/Appavang/livestream/playlist.m3u8
Persiana Game&Tech
documentary
http://51.210.199.25/hls/stream.m3u8
Persiana Sonati
music
http://51.210.199.59/hls/stream.m3u8
Ekran Movie
movies
http://159.69.58.154/ekran/ekrantv.m3u8
Persiana Korea
movies
http://51.210.199.14/hls/stream.m3u8
Manoto +1
entertain
http://159.69.58.154/manoto_plus1/manoto_plus.m3u8
Manoto +2
entertain
http://159.69.58.154/manoto_plus2/manoto2.m3u8
TehranTV 24
movies
http://51.210.227.132/hls/stream.m3u8
Persiana Comedy
movies
http://51.210.199.27/hls/stream.m3u8
Tin tv
entertain
https://bozztv.com/1gbw5/tintv/tintv/playlist.m3u8

**Comment by mh292929** [comment]
Health Media
documentary
https://j78dpkrjlq5r-hls-live.5centscdn.com/HMN/271ddf829afeece44d8732757fba1a66.sdp/playlist.m3u8
Azadi tv
Political
https://5b18be6964c2f.streamlock.net/m/azadah/playlist.m3u8
MTC
music
http://mellitv.tulix.tv/mellitv/myStream.sdp/playlist.m3u8
4 Kurd
music
http://51.210.199.28/hls/stream.m3u8
Eshare TV
movies
http://159.69.58.154/eshare/esharetv.m3u8
Bazme Asheghan
movies
https://5dc143936e9ae.streamlock.net/saeed_nl/saeed_nl/chunklist.m3u8
CPMTV 24
documentary
http://159.69.58.154/cpmtv/cpmtv.m3u8
Persian Film
movies
http://51.210.227.135/hls/stream.m3u8
1TvPersian
entertain
https://11778-1.b.cdn13.com/onetvpersia001.m3u8?bandwidth=3196&playlistlength=5&shift=0&v=2
Nour
Religious
https://cdn.videoevent.live:19360/elfaro4/elfaro4.m3u8
Persiana Classic
movies
http://51.210.199.26/hls/stream.m3u8
Persiana Iranian
movies
http://51.210.199.22/hls/stream.m3u8
AMG
movies
https://nl.livekadeh.com/hls2/AMG.m3u8
Faraz TV
movies
https://faraztv.net/hls/stream.m3u8

**Comment by mh292929** [comment]
Music 1
music
http://159.69.58.154/music1/music1_tv.m3u8
High Vision
music
https://streamer1.connectto.com/HIGHVISION_WEB_1205/tracks-v1a1/mono.m3u8
Kidstoons
kids
http://de2.martiatv.com:8080/hls/kidstvlive.m3u8
Parsiland
movies
http://vps.parsiland.net/hls/ParsiLand.m3u8
Payam Javan TV
movies
http://uni01rtmp.tulix.tv/kensecure/pjtv.stream/playlist.m3u8
AFN TV
entertain
https://bozztv.com/1gbw5/tintv2/tintv2/playlist.m3u8
Persian Bazar
other
https://stream.persiantv1.com/ptv1/playlist1/index.m3u8
ITC
entertain
https://dacastmmd.mmdlive.lldns.net/dacastmmd/f05d55e42dc746c8bd36edafbace7cc1/chunklist_b4628000.m3u8
Kalemeh TV
Religious
http://51.210.199.37/hls/stream.m3u8
Icc Tv
movies
http://mn-nl.mncdn.com/ezcdn-nownews/live_1/playlist.m3u8
Negah Tv
entertain
https://iptv.negahtv.com/negahtv/playlist2/index.m3u8
Khatereh Tv
movies
https://5caf24a595d94.streamlock.net:1937/8130/8130/playlist.m3u8
Omide Iran
entertain
http://51.210.199.38/hls/stream.m3u8

**Comment by mh292929** [comment]
Khaneh Honarmandan
movies
http://159.69.58.154/honarmandan/honarmandan.m3u8
Sat7 Pars
Religious
https://svs.itworkscdn.net/sat7parslive/sat7pars.smil/sat7parspublish/sat7pars_720p/chunks_dvr.m3u8
Payame Aramesh
Religious
http://51.15.133.13/hls/stream.m3u8
Salam TV
Religious
https://iptv.salaamtv.org/salaam/playlist.m3u8
Icnet TV
Religious
http://51.210.199.7/hls/stream.m3u8
BeitolAbbas Tv
Religious
http://live.beitolabbas.tv/live/beitolabbastv.m3u8
Imam Hussein TV
Religious
https://fa.imamhusseintv.com/live/ih1.m3u8
Wesal haq
Religious
https://stream.holol.com/w/whaq_360p/playlist.m3u8
marjaeyat
Religious
https://livefa.marjaeyattv.com/mtv_fa/playlist.m3u8
valiasr
Religious
https://ir13.livekadeh.com/hls2/valiasr.m3u8
Sepanj Tv
movies
http://51.210.199.30/hls/stream.m3u8
Persiana Sport
sport
http://159.69.58.154/varzesh3/varzesh3_tv.m3u8
Mono Tv
sport
http://51.210.227.137/hls/stream.m3u8
tapesh 2 s2
music
http://51.210.199.44/hls/stream.m3u8
hasti tv
https://live.hastitv.com/hls/livetv.m3u8

**Comment by mh292929** [comment]
please notify when you add in Persian language

**Comment by mh292929** [comment]
[radio.txt](https://github.com/iptv-org/iptv/files/6862345/radio.txt)
Hello. This file contains all Persian internet radios.
Please add it in the radio category as well.

**Reply by BellezaEmporium** [reply]
We're currently looking at your list of TV channels and we'll see what can we add.
We only focus on television channels, not radios. Therefore we cannot add this list. However, you may submit it to another project that may collect links of different radios.

**Reply by mh292929** [reply]
ok. all channels works. There are channels with other names and urls that I have submitted in your project that have just been renamed and I have sent you the latest list.

**Comment by mh292929** [comment]
hi. must i add channels without logo or you add these channels?

**Comment by mh292929** [comment]
sorry i can not found country for channels but i know all are persian language. please add to iptv. thanks.
